src/components/user/unavailability/unavailability.service.ts

```diff
@@ -18,7 +18,7 @@ export class UnavailabilityService {
   ) {}
 
   async create(
-    input: CreateUnavailability,
+    { userId, ...input }: CreateUnavailability,
```

Comment:
Same question

Reply:
Same approach as the Education service, just a few lines deeper in this function.

Reply:
Same response here 🤣

Reply:
Fixed!